src/components/pokedex.css

```diff
@@ -181,41 +182,42 @@
 .heart-button {
   position: absolute;
   top: -10px;
-  right: -200px; 
+  right: -190px;
   background: none;
   border: none;
   cursor: pointer;
+  border-radius: 50%;
+  padding: 10px;
   background-color: white;
 }
```

Comment:
minor thing but i expect you prefer this change ? 

<img width="350" alt="Captura de pantalla 2024-03-21 a las 21 03 25" src="https://github.com/leire16/master-inesdi-pokedex/assets/2211052/1f438603-41d4-42e7-8df3-a3a4a65d1da0">